packages/server/src/sdk/app/tables/internal/index.ts

```diff
@@ -47,7 +48,9 @@ export async function save(
   }
 
   // check for case sensitivity - we don't want to allow duplicated columns
-  const duplicateColumn = findDuplicateInternalColumns(table)
+  const duplicateColumn = findDuplicateInternalColumns(table, {
+    internalRowsAllowed: !!opts?.isImport,
```

Comment:
Can we name this to something which explains it a bit better? `internalRowsAllowed` doesn't really explain that its not checking for duplicates in the internal column names.

Reply:
Done as agreed!